<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Smoke" duration-ms="35220" started-at="2020-09-10T21:52:23 EDT" finished-at="2020-09-10T21:52:58 EDT">
    <groups>
      <group name="smoke">
        <method signature="AddEmployeesTest.addEmployee()[pri:0, instance:com.hrms.testcases.AddEmployeesTest@3a5ed7a6]" name="addEmployee" class="com.hrms.testcases.AddEmployeesTest"/>
        <method signature="LoginTest.validLogin()[pri:0, instance:com.hrms.testcases.LoginTest@6325a3ee]" name="validLogin" class="com.hrms.testcases.LoginTest"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Smoke" duration-ms="35220" started-at="2020-09-10T21:52:23 EDT" finished-at="2020-09-10T21:52:58 EDT">
      <class name="com.hrms.testcases.AddEmployeesTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.hrms.testcases.AddEmployeesTest@3a5ed7a6]" name="setUp" is-config="true" duration-ms="7564" started-at="2020-09-10T21:52:23 EDT" finished-at="2020-09-10T21:52:31 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="addEmployee()[pri:0, instance:com.hrms.testcases.AddEmployeesTest@3a5ed7a6]" name="addEmployee" duration-ms="9654" started-at="2020-09-10T21:52:31 EDT" finished-at="2020-09-10T21:52:40 EDT">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.hrms.utils.ExcelUtility.getCellData(ExcelUtility.java:46)
at com.hrms.utils.ExcelUtility.excelToListMap(ExcelUtility.java:58)
at com.hrms.testcases.AddEmployeesTest.addEmployee(AddEmployeesTest.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addEmployee -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.hrms.testcases.AddEmployeesTest@3a5ed7a6]" name="tearDown" is-config="true" duration-ms="650" started-at="2020-09-10T21:52:41 EDT" finished-at="2020-09-10T21:52:42 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrms.testcases.AddEmployeesTest -->
      <class name="com.hrms.testcases.LoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.hrms.testcases.LoginTest@6325a3ee]" name="setUp" is-config="true" duration-ms="7691" started-at="2020-09-10T21:52:42 EDT" finished-at="2020-09-10T21:52:49 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="validLogin()[pri:0, instance:com.hrms.testcases.LoginTest@6325a3ee]" name="validLogin" duration-ms="8159" started-at="2020-09-10T21:52:49 EDT" finished-at="2020-09-10T21:52:57 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.hrms.testcases.LoginTest@6325a3ee]" name="tearDown" is-config="true" duration-ms="650" started-at="2020-09-10T21:52:58 EDT" finished-at="2020-09-10T21:52:58 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrms.testcases.LoginTest -->
    </test> <!-- Smoke -->
  </suite> <!-- Smoke -->
</testng-results>
